scripts/bfb-install

```diff
@@ -48,18 +48,18 @@ usage ()
 {
   echo "Usage: $0 [options]"
   echo "Options:"
-  echo "  -a, --activate <0|1>           Activate the upgrade."
+  echo "  -a, --apply-lfwp <0|1>         Apply the upgrade. Default is 0: not to apply."
```

Comment:
What would be the command syntax for deferred upgrade and LFWP?
Do we use '--apply-lfwp 1' to differentiate these two cases?